libini2.h

```diff
@@ -20,4 +20,6 @@ int foreach_in_ini(const char *ini_file,
 
 int ini_unroll(const char *input, const char *output);
 
+void write_driver_name_to_ini(FILE *f, const char *driver_name);
```

Comment:
curios: will this be used later ?
because if it won't be, then maybe we could just leave it as static inside `libini2.c`

Reply:
Yes, it will be used later. I've mentioned this in the PR description.